sdk/communication/azure-communication-phonenumbers/src/main/java/com/azure/communication/phonenumbers/models/PhoneNumberError.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.phonenumbers.models;
+
+import com.azure.core.annotation.Fluent;
+import java.util.List;
+
+/** The Phone Number Services error. */
+@Fluent
+public final class PhoneNumberError {
+
+    private final String code;
+
+    private final String message;
+
+    private final String target;
+
+    private final List<PhoneNumberError> details;
+
+    private final PhoneNumberError innerError;
+
+    /**
+     * Constructs a new PhoneNumberError
+     * @param message The message of the original error
+     * @param code The error code
+     * @param target The target of the error
+     * @param details Additional details
+     * @param innerError The inner error
+     */
+    public PhoneNumberError(String message, String code, String target, List<PhoneNumberError> details, PhoneNumberError innerError) {
```

Comment:
innerError is not useful. The more expose, the more vulnerable we are in maintenance.